packages/react-scripts/scripts/start.js

```diff
@@ -94,6 +94,12 @@ function setupCompiler(host, port, protocol) {
 
     if (isSuccessful) {
       console.log(chalk.green('Compiled successfully!'));
+      if (messages.isFlowIntegrationErrorMessage) {
+        console.log(chalk.yellow('Flow checks were skipped.'));
+      }
+      if (stats.compilation.flowPassed) {
+        console.log(chalk.green('Flow checks have passed.'));
```

Comment:
I imagined that Flow users unaware of the extent of integration might think it didn't run. However, they'd probably make errors often enough to notice those are being reported. So maybe you're right.